react/features/device-selection/components/VideoInputPreview.js

```diff
@@ -114,6 +123,7 @@ class VideoInputPreview extends Component {
             <div
                 className = 'video-input-preview'
                 ref = { this._setRootElement }>
+                <FaceTracker />
```

Comment:
After looking some more at the rest of the code it seems like the FacePrompt should be the component that goes here. The FaceTracker should be separated from the prompt as it's the one doing the actual face detection. See more comments below for the FaceTracker!